src/routes/events/+page.ts

```diff
@@ -1,14 +1,51 @@
-import type { EventListData } from '$lib/EventListData';
+import type { EventListData } from '$lib/EventData';
+import type { LoadEvent } from '@sveltejs/kit';
 
-export function load(): EventListData {
-	return {
-		events: [
-			{
-				id: 1,
-				eventTitle: 'test',
-				eventDate: new Date(),
-				eventTime: new Date()
-			}
-		]
+export async function load({ fetch }: LoadEvent): Promise<EventListData | null> {
+	const url = 'http://localhost:8000/events';
```

Comment:
I would add from the beginning an env variable for the API endpoint so we don't have to care about it in the future as the codebase grows

Reply:
Yeah, I was being lazy, but good idea to get it handled now